grammar.js

```diff
@@ -1065,12 +1066,66 @@ module.exports = grammar({
       optional($.limit),
     ),
 
-    _table_expression: $ => seq(
-      $.table_expression,
+    relation_list: $ => seq(
+      choice(
+        $.relation,
+      ),
       repeat(
         seq(
           ',',
-          $.table_expression,
+          choice(
+            $.relation,
+          )
+        ),
+      ),
+    ),
+
+    _table_expression: $ => seq(
+      optional(
+        $.keyword_only,
+      ),
+      optional(
+        seq(
+          field('schema', $.identifier),
+          '.',
+        ),
+      ),
+      field('name', $.identifier),
+    ),
+
+    _inplace_insert: $ => seq(
+        '(',
+          $.keyword_values,
+          $.list,
+          optional(
+              repeat(
+              seq(
+                ',',
+                $.list,
+              ),
+            ),
+          ),
+          optional(
+            seq(
+              $.keyword_as,
+              field('table_alias', $.identifier),
+              $.list,
+            ),
+          ),
```

Comment:
you can specify a list of attributes after any relation (e.g. `select * from now() as n (blah)`) so this can be folded into the `relation` node. Also check out line 690 in this branch for `insert_expression`:

```
      optional(alias($._column_list_without_order, $.column_list)),
```

`list` is too generic, we allow column names and only column names here.